### PR TITLE
Add Dels demerit missions

### PR DESCRIPTION
**Pull request by lumbar527**
This PR adds missions to fine you when you have enough demerits from the into missions added in #12. Also adds more ship categories to the list of ships you can use for these missions.

**Checklist**
- [ ] Consider doing something other than fining.
- [ ] Test stuff.

### CODE REVIEW

File: data/Dels/Dels missions.txt [hunk 1]
```diff
@@ -925,6 +927,44 @@ mission "Dels Officer: Escort Convoy"
...
+		fine 100000
+		dialog `Suddenly, you recieve a message from navy command! It reads, "Officer <first>, you have over one hundered demerits. This is a great disapointment. Thus, you will be fined 100,000 credits."`
+
+mission "Dels Officer: Bad Officer 3"
+	landing
+	to offer
+		"dels officer: demerit count" > 500
+	on offer
+		fine 1000000
+		dialog `Suddenly, you recieve a message from navy command! It reads, "Officer <first>, you must STOP your infuriating behavior if you want to stay in the Kaote navy! As one final repirmand, you will be fined ONE MILLION credits."`
```
Comment (1010todd) [inline data/Dels/Dels missions.txt:955]: This dialog is very unprofessional and doesn't fit a proper government navy. Also 1 million isn't that and we don't need 4 stages, charge 1 millions on the 2nd offence and kick out on the third.

File: data/Dels/Dels missions.txt [hunk 2]
```diff
@@ -925,6 +927,44 @@ mission "Dels Officer: Escort Convoy"
...
+
+mission "Dels Officer: Bad Officer Final"
+	landing
+	to offer
+		"dels officer: demerit count" >= 1000
+	on offer
+		"reputation: Dels Kaote" -= 10
+		clear "license: Dels Officer"
+		clear "GW: Dels start: officer"
```
Comment (1010todd) [inline data/Dels/Dels missions.txt:964]:
```suggestion

```
Start probably shouldn't be removed, your past can't change.

File: data/Dels/Dels missions.txt [hunk 3]
```diff
@@ -925,6 +927,44 @@ mission "Dels Officer: Escort Convoy"
...
+	on offer
+		fine 1000000
+		dialog `Suddenly, you recieve a message from navy command! It reads, "Officer <first>, you must STOP your infuriating behavior if you want to stay in the Kaote navy! As one final repirmand, you will be fined ONE MILLION credits."`
+
+mission "Dels Officer: Bad Officer Final"
+	landing
+	to offer
+		"dels officer: demerit count" >= 1000
+	on offer
+		"reputation: Dels Kaote" -= 10
```
Comment (1010todd) [inline data/Dels/Dels missions.txt:962]:
```suggestion
		"reputation: Dels Kaote" <?= 1

```
Just make sure it doesn't drop below 0.

**Review by 1010todd — CHANGES_REQUESTED** [review]

File: data/Dels/Dels missions.txt (hunk 1 above)
Comment (lumbar527) [reply data/Dels/Dels missions.txt:955]: Would one mission where you get kicked out be better?

Comment (1010todd) [reply data/Dels/Dels missions.txt:955]: Not too sure what you meant by that.

File: data/Dels/Dels missions.txt [hunk 4]
```diff
@@ -925,6 +927,44 @@ mission "Dels Officer: Escort Convoy"
...
+	to offer
+		"dels officer: bad officer" >= 10
+	on offer
+		fine 10000
+		"dels officer: demerit count" += "dels officer: bad officer"
+		"dels officer: bad officer" = 0
+		dialog `Suddenly, you recieve a message from navy command! It reads, "Officer <first>, we are highly disappointed in you, as you have received many demerits. Because of this, you are being fined 10,000 credits."`
+
+mission "Dels Officer: Bad Officer 2"
+	landing
```
Comment (1010todd) [inline data/Dels/Dels missions.txt:942]: Might need `invisible` tag so the name don't show up in job board.

Comment (lumbar527) [reply data/Dels/Dels missions.txt:943]: Done, thanks!

File: data/Dels/Dels missions.txt (hunk 1 above)
Comment (lumbar527) [reply data/Dels/Dels missions.txt:958]: I meant something more like removing all missions except for a tweaked final mission (the one where you lose your license).

**Comment by lumbar527** [comment]
I'm probably not going to work on this any more.

**Pull request closed without merging**